protocol.c

```diff
@@ -3440,7 +3440,10 @@ void protocol (SOCKET socket_in, SOCKET socket_out, FTN_NODE *to, FTN_ADDR *fa,
     process_killlist (state.killlist, state.n_killlist, 's');
     inb_remove_partial (&state, config);
     if (to)
-      good_try (&to->fa, "CONNECT/BND", config);
+      if (config->remove_try_files == 0)
+        good_try (&to->fa, "CONNECT/BND", config);
+      else
+        remove_try (&to->fa, config);
```

Comment:
I'd suggest to move this check to the `good_try()` function, and there call either `write_try()` or `remove_try()` depending on `config->remove_try_files` option. It seems more consistent and readable than check this condition in `protocol.c`, because actually it was a good try regardless of `config->remove_try_files` setting.